luaui/Widgets/gui_unit_stats.lua

```diff
@@ -811,19 +816,25 @@ local function drawStats(uDefID, uID)
 	DrawTextBuffer()
 end
 
+function enableStats()
+	showStats = true
+end
```

Comment:
A new line between functions for aesthetics, also move before initialize and prepend with `local`. Not a blocker

Reply:
Fixed now. Thanks